platform/android/dependencies.gradle

```diff
@@ -21,8 +21,8 @@ ext {
             mapboxAndroidTelemetry : "com.mapbox.mapboxsdk:mapbox-android-telemetry:${mapboxServicesVersion}@aar",
 
             // mapzen lost
-            lost                   : 'com.mapzen.android:lost:3.0.0',
-
+            lost                   : 'com.mapzen.android:lost:3.0.1-SNAPSHOT',
```

Comment:
@tobrun Is there any way to anchor this dependency to a specific SNAPSHOT timestamp? I'm afraid that depending on a moving target can make troubleshooting harder.